apps/lfx-one/src/app/modules/dashboards/executive-director/components/revenue-impact-drawer/revenue-impact-drawer.component.html

```
@@ -0,0 +1,171 @@
+  </ng-template>
+
+  <!-- Content -->
+  <div class="flex flex-col gap-6 pb-2" data-testid="revenue-impact-drawer-content">
+    <!-- SECTION 1: Membership Growth Pipeline -->
+    <div class="flex flex-col gap-4" data-testid="revenue-impact-drawer-pipeline-section">
+      <h3 class="text-sm font-semibold text-gray-900">Membership Growth Pipeline</h3>
+      <div class="flex gap-4" data-testid="revenue-impact-drawer-stats">
+        <lfx-card styleClass="flex-1">
+          <div class="flex flex-col gap-1">
+            <span class="text-sm text-gray-500">Pipeline Influenced</span>
+            @if (data().pipelineInfluenced > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)